index.html

```diff
@@ -14,20 +14,22 @@
 	<header>
 		<h1>This is the Faith and Technology Initiative</h1>
 		<aside>A project by Calvin University students to explore the intersection of 
-			techological creation, and the ultimate creator.</aside>
+			techological creation and the ultimate Creator.</aside>
 	</header>
 	<hr />
 	<h2>Who are we, and what do we want?</h2>
-	<p>We are a collection of students, involved with the computer science and related 
-	departments at Calvin university. Our goal is to bring renewed interest from students and 
-	professionals alike regarding the roles that techology can play in a Christian life. </p>
-	<p>As a group, we have noticed the lack of conversation around computing and faith. Other
-	fields, such as philosophy, biology, and the arts, all have very pronounced intersections
-	with Christian life, but finding such meaning in more abstract fields (like computer
-	science, information systems, or mathematics) can be difficult.</p>
+	<p>We are a collection of students at Calvin University interested in the intersection 
+	between faith and technology. Our goal is to bring interest from students, faculty, and 
+	professionals regarding the roles that techology can play in a Christian life. </p>
+	<p>We believe there is deep connection between computing and faith. Though computer science
+	is more abstract than the natural sciences or the arts, we believe that the connections to Christian life
+	are significant and deserve attention. As Christians, we see that there is danger in blindly 
+	accepting new technology into our lives. As computer scientists, we see the value that 
+	technology is able to bring ot people's lives. We strive to encourage nuanced and ongoing conversations 
```

Comment:
bring **to**

Reply:
Fixed, along with a couple other typos in the .md. (We can easily switch back to HTML, but this is at least easier to preview, content-wise.)